nautobot/project-static/css/base.css

```diff
@@ -324,6 +324,10 @@ ul.nav-tabs li a:hover,
 .label a {
     color: inherit;
 }
+ul.software-image-hierarchy {
+    list-style-type: "↳ ";
+    padding-left: 18px;
+}
```

Comment:
If this is specific to the Device detail view, it *might* make sense to do as an inline style there instead of adding to base.css?